content/en/docs/setup/independent/install-kubeadm.md

```diff
@@ -213,6 +213,13 @@ mkdir -p /opt/cni/bin
 curl -L "https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/cni-plugins-amd64-${CNI_VERSION}.tgz" | tar -C /opt/cni/bin -xz
 ```
 
+Install crictl (required for kubeadm / Kubelet Container Runtime Interface (CRI))
```

Comment:
unify with the section bellow instead.

Reply:
add this:
```

CRICTL_VERSION="v1.11.1"
curl -L "https://github.com/kubernetes-incubator/cri-tools/releases/download/${CRICTL_VERSION}/crictl-${CRICTL_VERSION}-linux-amd64.tar.gz" | tar -C /opt -xz
```
^ also please fix `tar -C /opt -xz`

under:
```
 curl -L --remote-name-all https://storage.googleapis.com/kubernetes-release/release/${RELEASE}/bin/linux/amd64/{kubeadm,kubelet,kubectl}
 chmod +x {kubeadm,kubelet,kubectl}
```

Reply:
> my concern here is that we appear to be adding special treatment for a linux distro.
@neolit123, if you expand the code, you can see that this is being added to a Container Linux specific tab.

@discostur I'm fine whether you keep it as its own section or merge the crictl install with the one @neolit123 references.

I think it's clear as it's own section.
Please add a `mkdir -p /opt/bin` before the curl and write the tar contents to `/opt/bin`.

Reply:
sorry, i've missed the part where this is in a separate tab already:
`{{% tab name="Container Linux" %}}`

Reply:
@stealthybox @neolit123 added / corrected the instructions as you said:

https://github.com/kubernetes/website/pull/9798/commits/f5cd20bbbb6dce9248a5ebf14c07194be76a2acb